package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "express-timeout-handler",
-  "version": "2.2.0",
```

Comment:
:-(

Reply:
No no. We just need Oliver to add us as collaborators in NPM

Reply:
We are not moving it to @debitoor

Reply:
@Oligrand Help :)

Reply:
You are added now @jonatanpedersen 🚀

Reply:
Yay, thanks @Oligrand. Closing this PR then :)

Reply:
Cool, and nice to see this module is getting some love! @hilleer

Reply:
@Oligrand I still don't have access

Reply:
Yay, it works now :)

Reply:
that's weird, I can see you:

![image](https://user-images.githubusercontent.com/7648431/60658041-ae486e00-9e52-11e9-95e2-2a35697dacdc.png)

Reply:
Ah ok it works now lol